pom.xml

```diff
@@ -160,6 +160,8 @@
         <mysql.version>8.0</mysql.version>
         <mysql-jdbc.version>8.3.0</mysql-jdbc.version>
         <postgresql.version>15</postgresql.version>
+        <autora-postgresql.version>15.3</autora-postgresql.version>
```

Comment:
`aurora-postgresql`?

Reply:
Thanks, fixed in next commit I just pushed. Builds looked good, before, waiting for another green build before marking it ready.